examples/hybrid/driver.jl

```diff
@@ -227,3 +219,27 @@ if config.parsed_args["check_precipitation"]
         @test maximum(sol.prob.p.precomputed.ᶜcloud_fraction[colidx]) <= FT(1)
     end
 end
+
+# Visualize the solution
```

Comment:
I wonder if it is better to move it back, as we may still want to visualize a simulation even if the conservation check fails. Not a big issue though, so up to you. The artifact failure should be fixed soon.

Reply:
Ahh, good point! The plotting kept failing for all sorts of reasons, which is why I moved it. I could add `skip=true` to `@test` for now? Hopefully the conservation checking will be revamped soon anyway with a more sophisticated solution.

Reply:
hmm, maybe let's keep it as it is then. I'm afraid that people will merge PRs without paying attention to whether this check fails if we set skip=true.